### PR TITLE
Added Docstrings to the constant_pad function

### PR DESCRIPTION
**Pull request by Klins101**
- [ ] #1672

### CODE REVIEW

File: ivy/functional/ivy/manipulation.py [hunk 1]
```diff
@@ -747,12 +747,90 @@ def constant_pad(
...
+    Using :code:`ivy.Container` instance method:
+    >>> x = ivy.Container(a=ivy.array([1., 2., 3.]), \
+                          b=ivy.array([4., 5., 6.]))
+    >>> y = ivy.constant_pad(x, (2, 3), (5., 6.))
+    >>> print(y)
+    {
+        a: ivy.array([5. 5. 1. 2. 3. 6. 6. 6.]),
+        b: ivy.array([5. 5. 4. 5. 6. 6. 6. 6.])
+    }
+
```
Comment (Aarsh2001) [inline ivy/functional/ivy/manipulation.py:833]: Hi @Klins101, the formatting guide has changed a little bit. We no longer add all examples in the ivy docstring, rather we add it to their implementation specific docsrings. For example,  ``` instance method examples ``` for container will go in the ```container/manipulation/constant_pad``` function docstring. Kindly refer to the guide [here](https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html).

Also, while calling an instance method on an ```array``` or a  ```container```, we pass in that value as self and call the function with a ```dot``` operator. For example -:
```
x = ivy.array([1., 2., 3., 4.])
y = x.constant_pad((2,3), (5,6))

```

Kindly refer to the changes above 🙂

Comment (Klins101) [reply ivy/functional/ivy/manipulation.py]: Noted with thanks. I've made the required changes.

**Comment by Aarsh2001** [comment]
Hey @Klins101, can you please check your implementation, the docstring tests should pass🙂

File: ivy/container/manipulation.py [hunk 2]
```diff
@@ -333,6 +333,22 @@ def constant_pad(
...
+        with minimal changes.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([1, 2, 3]), \
+                              b=ivy.array([4, 5, 6]))
+        >>> y = x.constant_pad((2, 3), (5, 6))
+        >>> print(y)
+        {
+            a: ivy.array([5, 5, 1, 2, 3, 6, 6, 6,]),
```
Comment (Aarsh2001) [inline ivy/container/manipulation.py:348]: Hi, so the ```,``` at the end, inside this array, is causing the tests to fail. Can you please remove it?

**Comment by Aarsh2001** [comment]
Hey @Klins101, so i have requested a few changes above. Also, please keep in bind about the repetitiveness of the examples. So for instance the functional examples, are similar to the instance method examples for both ```array``` and ```container```. Can you please change either of them, thanks🙂

**Comment by Aarsh2001** [comment]
Hey @Klins101, I'm closing this PR as of now. Feel free to re-open when you're ready with the changes 🙂

**Pull request closed without merging**